.drone.star

```diff
@@ -1804,6 +1804,7 @@ def e2eTestsOnKeycloak(ctx):
     e2e_Keycloak_tests = [
         "journeys",
         "admin-settings/users.feature:20",
+        "admin-settings/spaces.feature",
```

Comment:
good

Reply:
Yes, I try to run full, but it fails. There is problem on afterall during clean-up which fails to delete the users from oCIS-web. I will cover it another PR. May be after that we can include other feature file for keycloak.

Since there is issue on group creation for keycloak  [issue](https://github.com/owncloud/ocis/issues/5538). So, I think we cannot include the tests related to the group.

Reply:
@ScharfViktor I think **user.feature** file contains scenarios which change users ceredentials like **name, role and password**. If we run e2e scenario in **user.feature** which changes user **name, role and password** through web-UI then our tests fail because ocis side changes on user **name , password , role** will not sync with keycloak. Is this actual feature of ocis or a bug?
If **unable to change user name, role, password from web**  is a feature then  **change quota** is only e2e scenario we can run from **user.feature** file